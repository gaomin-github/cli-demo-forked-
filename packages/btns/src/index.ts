"use strict";
export { NormalButton } from './normal-button';