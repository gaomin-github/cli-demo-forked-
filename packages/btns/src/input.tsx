import React from 'react';
export const NormalInput = () => {
  const [inputMsg, updateInputMsg] = React.useState('');
  const updateChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    updateInputMsg(e.target.value);
    // console.log('updateChange', e);
  };

  return (
    <div>
      <input onChange={updateChange} />
      {/* <div dangerouslySetInnerHTML={inputMsg}></div> */}
      {inputMsg}
      <a href="http://localhost: :51768/?path=/story/btns--base-demo-1&javascript:alert('1')">
        thisis
      </a>
    </div>
  );
};
