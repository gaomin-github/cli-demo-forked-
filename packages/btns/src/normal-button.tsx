import React from 'react';
// import Tool from '@cli-demo-pj/tools';

export const NormalButton = () => {
  const handleBtnClick = () => {
    // Tool.printLog('normalButton');
  };

  return (
    <div onClick={handleBtnClick} className="normal-button">
      normal button
    </div>
  );
};
