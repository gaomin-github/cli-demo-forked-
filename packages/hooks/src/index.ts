export { useMergedState } from './use-merged-state';