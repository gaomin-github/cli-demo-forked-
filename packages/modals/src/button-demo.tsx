import React from 'react';

export const ButtonDemo = () => {
  return <div>button demo</div>
}