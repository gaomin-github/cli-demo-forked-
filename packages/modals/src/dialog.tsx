import React from 'react';
import { NormalButton } from '@cli-demo-pj/btns/src/normal-button';
import { ButtonDemo } from './button-demo';

export const Dialog = () => {
  return (
  <div>
    dialog
    <div>
      <NormalButton />
    </div>
    <div>
      <ButtonDemo/>
    </div>
  </div>
  );
}