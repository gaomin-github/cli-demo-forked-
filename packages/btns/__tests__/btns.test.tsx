'use strict';

import { render } from '@testing-library/react';
import React from 'react';
import { NormalButton } from '../src';

describe('btns', () => {
  it('normalButton', () => {
    const { container } = render(<NormalButton />);
    expect((container as Element).querySelector('.normal-button')).not.toBeNull();
    expect(container).toMatchSnapshot();
  });
});
