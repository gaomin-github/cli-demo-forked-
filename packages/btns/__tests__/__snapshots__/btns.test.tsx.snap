// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`btns normalButton 1`] = `
<div>
  <div
    class="normal-button"
  >
    normal button
  </div>
</div>
`;
